docs/api.md

```diff
@@ -1237,6 +1251,12 @@ The element will be passed as the first argument to `pageFunction`, followed by
 This method scrolls element into view if needed, and then uses [page.mouse](#pagemouse) to hover over the center of the element.
 If the element is detached from DOM, the method throws an error.
 
+#### elementHandle.tap()
+- returns: <[Promise]> Promise which resolves when the element is successfully tapped. Promise gets rejected if the element is detached from DOM.
+
+This method scrolls element into view if needed, and then uses [touchScreen.tap](#touchscreentapx-y) to tap in the center of the element.
```

Comment:
can we explain what's a tap?

Reply:
A concise and effective definition: https://api.jquerymobile.com/tap